template/build/webpack.prod.conf.js

```diff
@@ -1,4 +1,5 @@
 var path = require('path')
+var glob = require('glob-all')
```

Comment:
Nope, for include few files or subdirectories need glob-all. I tried it, searched also.